Ghidra/Framework/Generic/src/main/java/generic/concurrent/QProgressListener.java

```diff
@@ -46,7 +45,7 @@
 	 * @param totalCount the total number of items that have been submitted to the ConcurrentQ
 	 * @param completedCount the total number of items that completed processing.
 	 */
-	void taskEnded(long id, I Item, long totalCount, long completedCount);
+	void taskEnded(long id, I item, long totalCount, long completedCount);
```

Comment:
I changed the actual parameter name instead of the docstring so that it conforms to the Ghidra Java Coding Standards section 2.1